include/forum.php

```diff
@@ -0,0 +1,190 @@
+<?php
+
+/**
+ * @file include/forum.php
+ * @brief Functions related to forum functionality *
+ */
+
+/**
+ * @brief This class handles functions related to the forum functionality
+ */
+class forum {
```

Comment:
OK I renamed it to ForumManager.
How is it with the name of the Methods? I have read somewhere that method should be also named CamelCase (e.g countUnseenItems() instead of count_unseen_items()   )